agency_client/Cargo.toml

```diff
@@ -20,30 +20,10 @@ reqwest = "0.9.5"
 regex = "1.1.0"
 rmp-serde = "0.13.7"
 rust-base58 = "0.0.4"
-indy = "1.16.0-post-59"
+indy = { git = "https://gitlab.com/evernym/verity/vdr-tools.git", rev = "59e1ecc21c9a067" }
```

Comment:
It's a pity the gitlabl repo is not versioned using tags (git revisions can't be ordered / compared easily)

Reply:
True, I believe they will start version it eventually though.